controllers/articles.controllers.js

```diff
@@ -11,4 +15,14 @@ const getArticlesById = (req, res, next) => {
     });
 };
 
-module.exports = { getArticlesById };
+const getAllArticles = (req, res, next) => {
+  selectAllArticles()
+    .then((data) => {
+      res.status(200).send(data);
+    })
```

Comment:
Don't you think sending the response back as data is a little generic 🤔 ?  
Would sending back on a key of articles make it more readable 💭

Reply:
Done